_episodes/21-automatically-testing-software.md

```diff
@@ -90,7 +90,7 @@ a common term we use to refer to sets of tests - that we'll use for our test wri
 
 ~~~
 $ git switch develop
-$ git branch -c test-suite
+$ git switch -c test-suite
```

Comment:
Whilst `git branch -c` is valid, it is never a command I have used. Using `-c` with switch is certainly preferred.